docroot/sites/settings/cgov_core.settings.php

```diff
@@ -37,6 +37,7 @@
       $domain = $site_name;
     }
   }
+  $settings['cgdp.cache_url_host'] = 'https://' . $domain;
```

Comment:
@navsunka (cc: @blairlearn ) This is probably why testing is failing on www-dev-ac. That www-dev/test/int-ac, nor odes, would satisfy the check on line 31, `if ($is_acsf_env && $acsf_db_name) {`.

So cgdp.cache_url_host will not be set when you do the code in the hook:
```
   // The value for cgdp.cache_url_host is computed in cgov_core.settings.php.
    $host = Settings::get('cgdp.cache_url_host');
    if (empty($host)) {
      return;
    }
```

Which means the cache will not be cleared.

Reply:
if $host is empty, we will then fetch the current base URL which will work for lower environments.